baremetalpxeserver/ansible/baremetalpxeserver.py

```
@@ -125,11 +118,23 @@
 else:
     error("unsupported OS!")
 
+# name: install virtualenv
+py_version = get_virtualenv_python_version(virtenv_path, host_post_info)
+if py_version and not py_version.startswith("3.11"):
+    command = "rm -rf %s" % virtenv_path
+    run_remote_command(command, host_post_info)
```

Ruff (0.15.2)

[error] `run_remote_command` may be undefined, or defined from star imports

(F405)